sdk/core/System.ClientModel/tests/client/ModelReaderWriter/ServiceModels/SystemData.Serialization.cs

```diff
@@ -1,10 +1,6 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-// <auto-generated/>
```

Comment:
For ClientModel-based client models, we need to either remove this line or explicitly set `#nullable enable` per https://learn.microsoft.com/en-us/dotnet/csharp/nullable-migration-strategies